pulser/simulation/simulation.py

```diff
@@ -106,7 +106,7 @@ def __init__(
         self._sampling_rate = sampling_rate
         self._qid_index = {qid: i for i, qid in enumerate(self._qdict)}
         self._collapse_ops: list[qutip.Qobj] = []
-        self._times = self._adapt_to_sampling_rate(
+        self.sampling_times = self._adapt_to_sampling_rate(
```

Comment:
So we're not going with `_sampling_times` so the user can have access to them, is that right?

Reply:
Yes, although it could be confusing. Notice this doesn't include the final instant, because it's just the indices from the pulse sequence divided by 1000 and adapted to sampling rate.